react/tic-tac-toe/src/app/screens/Login/components/LoginForm/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { func, string } from 'prop-types';
+import cn from 'classnames';
+
+import Button from '~components/Button';
+
+import Input from '~components/Input';
+
+import { ReactComponent as Logo } from '../../../../assets/error.svg';
+
+import styles from './styles.module.scss';
+
+import { validate } from '~utils/validations';
+
+function LoginForm({ handleSubmit, error }) {
+  return (
+    <form className={styles.form} onSubmit={handleSubmit}>
+      <Field type="text" component={Input} name="email" autocomplete="off" label="Email address" />
```

Comment:
Use `type="email"` for mail inputs, it will guide the user's OS to work accordingly. For example, in mobile devices, the keyboard will come with the `@` to make it easier to complete an email address.